@cypress/webpack-preprocessor 5.13.0 causes module error

### Current behavior

After updating from *@cypress/webpack-preprocessor* `5.12.2` to version `5.13.0` this error appears
```bash
Error: Cannot find module 'md5'
Require stack:
- /Users/redacted/redacted/react-shop/node_modules/@cypress/webpack-preprocessor/dist/lib/utils.js
- /Users/redacted/redacted/react-shop/node_modules/@cypress/webpack-preprocessor/dist/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:955:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/Users/redacted/Library/Caches/Cypress/10.9.0/Cypress.app/Contents/Resources/app/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:803:27)
    at Module.require (node:internal/modules/cjs/loader:1021:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object. (/Users/redacted/redacted/react-shop/node_modules/@cypress/webpack-preprocessor/dist/lib/utils.js:32:29)
    at Module._compile (node:internal/modules/cjs/loader:1119:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1173:10)
    at Object.require.extensions. [as .js] (/Users/redacted/Library/Caches/Cypress/10.9.0/Cypress.app/Contents/Resources/app/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:997:32)
    at Function.Module._load (node:internal/modules/cjs/loader:838:12)
    at Module.require (node:internal/modules/cjs/loader:1021:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object. (/Users/redacted/redacted/react-shop/node_modules/@cypress/webpack-preprocessor/dist/index.js:33:31)
    at Module._compile (node:internal/modules/cjs/loader:1119:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1173:10)
    at Object.require.extensions. [as .js] (/Users/redacted/Library/Caches/Cypress/10.9.0/Cypress.app/Contents/Resources/app/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:997:32)
    at Function.Module._load (node:internal/modules/cjs/loader:838:12)
    at ModuleWrap. (node:internal/modules/esm/translators:170:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
```

`node_modules/@cypress/webpack-preprocessor/dist/lib/utils.js:32:29`
```javascript
  30 | var os = __importStar(require("os"));
  31 | var path_1 = __importDefault(require("path"));
> 32 | var md5_1 = __importDefault(require("md5"));
     |                             ^
  33 | var bluebird_1 = __importDefault(require("bluebird"));
  34 | var fs_extra_1 = __importDefault(require("fs-extra"));
  35 | function createDeferred() {
  36 |     var resolve;
```

### Desired behavior

Cypress should work without an issue.

### Test code to reproduce

Run Cypress `10.8.0` or newer with @cypress/webpack-preprocessor `5.13.0`

### Cypress Version

10.8.0

### Node version

v18.8.0

### Operating System

macOS 12.6

### Debug Logs

_No response_

### Other

_No response_

--- a/npm/webpack-preprocessor/package.json
+++ b/npm/webpack-preprocessor/package.json
@@ -25,6 +25,7 @@
     "fs-extra": "^10.1.0",
     "loader-utils": "^2.0.0",
     "lodash": "^4.17.20",
+    "md5": "2.3.0",
     "webpack-virtual-modules": "^0.4.4"
   },
   "devDependencies": {
@@ -50,7 +51,6 @@
     "eslint-plugin-mocha": "8.1.0",
     "fast-glob": "3.1.1",
     "find-webpack": "1.5.0",
-    "md5": "2.3.0",
     "mocha": "^7.1.0",
     "mockery": "2.1.0",
     "proxyquire": "2.1.3",

